R/pkg/R/utils.R

```diff
@@ -689,3 +689,7 @@ getSparkContext <- function() {
   sc <- get(".sparkRjsc", envir = .sparkREnv)
   sc
 }
+
+is_master_local <- function(master) {
+  grepl("^local(\\[[0-9\\*]+\\])?$", master, perl = TRUE)
```

Comment:
I think you didn't see my comment here: https://github.com/apache/spark/pull/14258#discussion_r72333901